src/main/java/io/beekeeper/pingdom/resources/PingdomHealthResource.java

```diff
@@ -101,6 +104,9 @@ public PingdomHealth get(@QueryParam("key") String key, @QueryParam("severity")
                                 .append("Reason: ").append(result.getMessage()).append('\n')
                                 .append("Exception: ").append(result.getError() != null ? result.getError().getMessage() : null)
                                 .append('\n');
+                        if (log.isInfoEnabled()) {
```

Comment:
It would make sense to make this a Warning instead of an info

Reply:
Changed by d82fd58